packages/wallets/src/features/cfd/flows/MT5/AvailableMT5AccountsList/__test__/AvailableMT5AcountsList.spec.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { useActiveWalletAccount, useMT5AccountsList, useTradingPlatformStatus } from '@deriv/api-v2';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
hey hey @lubega-deriv 
i know that we are trying to migrate to use userEvent because its more like user behaviour instead of fireEvent, could we replace it?

Reply:
Updated